src/ORM/Connect/MySQLDatabase.php

```diff
@@ -360,10 +360,7 @@ public function transactionEnd($chain = false)
             return false;
         }
         --$this->transactionNesting;
-        if ($this->transactionNesting <= 0) {
-            $this->transactionNesting = 0;
-            $this->query('COMMIT AND ' . ($chain ? '' : 'NO ') . 'CHAIN');
-        }
```

Comment:
I feel like maybe this logic should remain, could explain the test breakages

Reply:
I added this back locally and re-tested and it fixes the new failures, pushed a fix to add this back

Reply:
derp